### ISSUE TITLE
Bonemeal Usage

### ISSUE THREAD

**Issue by essorencz** [Shrimpstress/cc-tweaked-fabric#15]
Minecraft 1.16
Im using farming turtle but it cant place a bonemeal on any crop.
![image](https://user-images.githubusercontent.com/22052729/96013467-94fbe300-0e45-11eb-8cfe-0d2771905781.png)
Any ideas that could fix this ? We are using Fabric with 82 other mods on it.

**Comment by EmmaKnijn** [comment]
Going to check this right now

**Comment by EmmaKnijn** [comment]
Yep doesn't work here, here is a little gif of what I'm trying to do 

![image](https://user-images.githubusercontent.com/64010592/96866040-06055100-146b-11eb-9f70-0eeed5d39b5f.png)


Minecraft / Fabric Version: 1.16.3 / 0.10.1+build.209
